services/galley/src/Galley/External/LegalHoldService.hs

```diff
@@ -56,9 +57,12 @@ checkLegalHoldServiceStatus fpr url = do
 requestNewDevice :: TeamId -> UserId -> Galley NewLegalHoldClient
 requestNewDevice tid uid = do
     resp <- makeLegalHoldServiceRequest tid reqParams
-    case decode (responseBody resp) of
-        Nothing -> throwM legalHoldServiceBadResponse
-        Just client -> pure client
+    case eitherDecode (responseBody resp) of
+        Left e -> do
+            lg <- view applog
+            Logger.err lg . msg $ "Error decoding NewLegalHoldClient: " <> e
```

Comment:
```suggestion
            Logger.info lg . msg $ "Error decoding NewLegalHoldClient: " <> e
```

I think `warn` would also be ok.  I think `err` is for things that seriously go wrong inside the server.